Cargo.toml

```diff
@@ -46,7 +46,8 @@ instant = { version = "0.1", features = ["wasm-bindgen"] }
 once_cell = "1.12"
 log = "0.4"
 serde = { version = "1", optional = true, features = ["serde_derive"] }
-raw-window-handle = "0.5.0"
+raw_window_handle_05 = { package = "raw-window-handle", version = "0.5" }
```

Comment:
Nit: Maybe use `raw_window_handle` for the new version, and `raw_window_handle_04` for the old?

Reply:
yeah, makes sense

Reply:
fixed